src/components/ContentCard/ContentCard.tsx

```diff
@@ -13,26 +14,78 @@ type ContentCardProps = {
   title: string;
   author: string;
   image: string;
+  authorImage: string;
   pressFunction: (event: GestureResponderEvent) => void;
 };
 
 function ContentCard({
   title,
   author,
   image,
+  authorImage,
   pressFunction,
 }: ContentCardProps) {
+  const saveStory = () => {
+    console.log("testing '+' icon does something for story " + title);
+  };
+
   return (
     <Pressable onPress={pressFunction}>
       <View style={styles.contentCard}>
-        <Image style={styles.image} source={{ uri: image }} />
+        <View style={{ flexDirection: 'row' }}>
+          <Image style={styles.image} source={{ uri: image }} />
+          <Image
+            style={styles.authors}
+            source={{
+              uri: 'https://gwn-uploads.s3.amazonaws.com/wp-content/uploads/2023/12/10090438/Angela-Trakoshis-100x100.jpg' /* change image for proper author when we can pull author images properly */,
+            }}
+          />
+        </View>
+
         <View style={styles.textContainer}>
           <Text style={[globalStyles.h3, styles.title]} numberOfLines={1}>
             {title}
           </Text>
-          <Text style={globalStyles.body1} numberOfLines={1}>
-            {author}
-          </Text>
+          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+            <Text style={globalStyles.subtext} numberOfLines={1}>
+              By{'  '}
```

Comment:
The "By" text should isn't in globalStyles, and should follow these styles: 
<img width="239" alt="Screenshot 2024-03-12 at 11 00 43 PM" src="https://github.com/calblueprint/girls-write-now/assets/34043950/1a8ecb01-7084-47b5-9db8-c1a28a446c9d">